projects/adrv9009/Makefile

```diff
@@ -1,6 +1,13 @@
 IIOD ?= n
 include ../../tools/scripts/generic_variables.mk
 
-include src.mk
+# Uncomment to select the profile:
+#PROFILE = tx_bw100_ir122p88_rx_bw100_or122p88_orx_bw100_or122p88_dc122p88
+#PROFILE = tx_bw200_ir245p76_rx_bw200_or245p76_orx_bw200_or245p76_dc245p76
+PROFILE = tx_bw400_ir491p52_rx_bw200_or245p76_orx_bw400_or491p52_dc245p76
+
+#include arm_src.mk
+# Uncomment the line below and comment the line above to build the ADRV9009 project for MB
+include mb_src.mk
```

Comment:
this needs to be either decided in the Makefile using `util.tcl get_arch` or through some variable that's passed at commandling when doing `make`

Reply:
Done